schema/url/computes/correlation.raml

```diff
@@ -2,9 +2,21 @@
 
 types:
 
+  FeaturePrefilterThresholds:
+    additionalProperties: false
+    properties:
+      proportionNonZero: 
+        type: number # should be between 0 and 1 inclusive
+      variance:
+        type: number # should be 0 or greater
+      standardDeviation:
+        type: number # should be 0 or greater
+
   CorrelationComputeConfig:
     properties:
       correlationMethod: CorrelationMethod
+      prefilterThresholds: FeaturePrefilterThresholds
```

Comment:
should be optional? We'd have basic defaults i'd think?

Reply:
When I tested they are functionally optional, but I agree the API should be explicit